### ISSUE TITLE
Logo location is messed up in 4.3 

### ISSUE THREAD

**Issue by RakibRyan** [Redot-Engine/redot-engine#640]
### Tested versions

Godot v4.3.stable  

### System information

Fedora Linux 40 (Workstation Edition) - Wayland - GLES3 (Compatibility) - Mesa Intel(R) Graphics (RKL GT1) - 11th Gen Intel(R) Core(TM) i5-11400 @ 2.60GHz (12 Threads)

### Issue description

https://github.com/user-attachments/assets/69583c23-86c8-42b1-a3b1-ff64e35e39ba

### Steps to reproduce

I'm not sure. Maybe it has to do with linux or big monitors.

### Minimal reproduction project (MRP)

N/A

**Comment by RakibRyan** [comment]
Commenting here because I think this bug might appear in redot as well

**Comment by Starkium** [comment]
did you change the logo? why is it purple?

**Comment by RakibRyan** [comment]
Yes I did use a different splash screen but the same project same settings seems to work fine on windows. Here's I tried removing the splash it still shows up like this.

**Comment by RakibRyan** [comment]
https://github.com/user-attachments/assets/07b99f15-8847-40f7-93e4-e1bb95a5fbb1

**Comment by RakibRyan** [comment]
Just to be sure I did test it out in a virtual windows setup. This is a godot-linux issue.

https://github.com/user-attachments/assets/a74a6727-ee35-4ce6-bc32-64b1809e9703

**Comment by BradGaddis** [comment]
Are you sure that it's not a Gnome or a Fedora issue with Godot? I am running Linux Mint and don't have the same issue.

**Comment by jawbroken** [comment]
why are you reporting real bugs here? the guy who made this fork can't code or fix them

**Comment by Arxari** [comment]
Yeah was about to say, the issue is much better of in the main repo where it can actually be addressed. I would report it but I don't know what your logic behind the splashscreen is so I can't.

**Comment by RakibRyan** [comment]
Just to see if I get the same issue in mint like you said. It seems like I get this issue when i open a project that was made in windows and later opened in linux. I'm not sure if it's a bug or something else is going on. 
@jawbroken @Arxari  you're welcome to report it yourself if you feel that strong about something. I was blocked from there for asking valid questions. 

https://github.com/user-attachments/assets/d27418ca-7fe2-42de-ae40-6ea6997b9d04

**Comment by jawbroken** [comment]
can we get some information about what "valid questions" you were asking in order to get blocked on github, thank you

**Comment by jawbroken** [comment]
think my posts disappeared somehow. possible bug? 🤔 just reiterating that this project is a grievance-based scam and the real project (with people who can actually program a game engine) is [here](https://github.com/godotengine/godot)

**Comment by AwesomeParley** [comment]
> think my posts disappeared somehow. possible bug? 🤔 just reiterating that this project is a grievance-based scam and the real project (with people who can actually program a game engine) is [here](https://github.com/godotengine/godot)

Nope! This is a real engine and isn't a scam, and that post of yours that disappeared wasn't a bug. The post was deleted due to it being irrelevant to the open issue. 

Here's the reason I say it's not a scam. The Redot team is (so far) [not accepting any money](https://x.com/RedotEngine/status/1841611449858208198), and will not for many months. This fork was created due to recent events that were very political and this project aims to [keep all politics out of it](https://x.com/RedotEngine/status/1842717403701985323). If you want to have more info, check the [Official X Redot Profile](https://x.com/RedotEngine) and also maybe check the [Discord](https://discord.gg/redot). 

The reason that this is posted in the Redot issues, is because of a Godot logo misplacement that may affect Redot as well due to the recent fork. [RakibRyan](https://github.com/RakibRyan) sent [this message](https://github.com/Redot-Engine/redot-engine/issues/640#issuecomment-2402455142) talking about the issue in regards to Redot:
> Commenting here because I think this bug might appear in redot as well

So if the issue persists in Redot, we can fix it. 

What we have figured out about this specific bug is that it does exist in the Linux version of Godot, and seems to not exist in the Windows version, and it isn't because of editing the logo to be that specific color.

**Comment by digimbyte** [comment]
bug seems to be related to capturing the screen size of the window, could be internal with the renderer stack.
is this a regression from 4.2 to 4.3?

**Comment by SkogiB** [comment]
Leaving this open as it is a Godot bug, but NOT a Godot 4.4dev bug as is the case with other recent bugs that we expect them to fix soon. Might be one we can address if they don't have active PRs for it.

**Comment by Spartan322** [comment]
Given this appears to be a Godot related bug, does this have a related Godot issue?

**Comment by RakibRyan** [comment]
Nope. Couldn’t report to godot as some wise guy banned me on godot github repo.

**Comment by Arxari** [comment]
Why?

**Comment by RakibRyan** [comment]
The gist of it is I asked what was going on with godot. And if politics is the only identity of godot. Anyways it’s not relevant anymore. So, I will drop it here. You are welcome to report it.

**Comment by Arctis-Fireblight** [comment]
Can someone from the @Redot-Engine/bug-hunter team confirm if this is still an issue with Redot?
I havent observed this issue with any of my Linux builds but I am wanting to make sure this isnt specifically an Intel / OpenGL issue before I mark this as closed.

**Comment by Arctis-Fireblight** [comment]
Never mind, I just observed this with OpenGL while attempting to confirm another bug.
This is still present in 4.4.Alpha3

**Comment by AliYar-Khan** [comment]
@Arctis-Fireblight my experience with 4.3 stable on linux mint

It doesn't seem to be centered

<img width="1600" height="860" alt="Image" src="https://github.com/user-attachments/assets/f733a99b-1e74-4533-ada7-2b42dcd1786a" />

**Comment by Arctis-Fireblight** [comment]
> [@Arctis-Fireblight](https://github.com/Arctis-Fireblight) my experience with 4.3 stable on linux mint
> 
> It doesn't seem to be centered
> 
> <img alt="Image" width="1600" height="860" src="https://private-user-images.githubusercontent.com/37889962/482201623-f733a99b-1e74-4533-ada7-2b42dcd1786a.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTYyMzU3ODUsIm5iZiI6MTc1NjIzNTQ4NSwicGF0aCI6Ii8zNzg4OTk2Mi80ODIyMDE2MjMtZjczM2E5OWItMWU3NC00NTMzLWFkYTctMmI0MmRjZDE3ODZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODI2VDE5MTEyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMwZDE0ODZkODRhYzc1NGYyMGQ3Yjg0MGM5NzRjZDBmMjM5ZGFjNTgyYTM4ZGMyYjlmNDUwZTc0MDczMDE2N2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._TDBHi8klRS0VL7QQn0fgnSG2tymyW91evIF28Aq6xA">

Thanks @AliYar-Khan , is this OpenGL or Vulkan?

**Comment by AliYar-Khan** [comment]
> > [@Arctis-Fireblight](https://github.com/Arctis-Fireblight) my experience with 4.3 stable on linux mint
> > It doesn't seem to be centered
> > <img alt="Image" width="1600" height="860" src="https://private-user-images.githubusercontent.com/37889962/482201623-f733a99b-1e74-4533-ada7-2b42dcd1786a.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTYyMzU3ODUsIm5iZiI6MTc1NjIzNTQ4NSwicGF0aCI6Ii8zNzg4OTk2Mi80ODIyMDE2MjMtZjczM2E5OWItMWU3NC00NTMzLWFkYTctMmI0MmRjZDE3ODZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODI2VDE5MTEyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMwZDE0ODZkODRhYzc1NGYyMGQ3Yjg0MGM5NzRjZDBmMjM5ZGFjNTgyYTM4ZGMyYjlmNDUwZTc0MDczMDE2N2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._TDBHi8klRS0VL7QQn0fgnSG2tymyW91evIF28Aq6xA">
> 
> Thanks [@AliYar-Khan](https://github.com/AliYar-Khan) , is this OpenGL or Vulkan?

how to check it?
